code-challenges.test.js

```diff
@@ -25,6 +25,21 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// Psuedo:
+// Create a new test using the descrive keyword
+// designate the name of our tested function inside the '' within the test.
+// Make sure to remember parenthesis for the call back function.
+// Use the 'it' keyword to describe inside of a string what the function will do.
+// Create 3 different expect statements for the outcomes of the objects below.
+
+describe("divBYThree", () => {
+    it("function will return string interpelation the number is divisible by three, or the number is not divisible by three.", () => {
+      expect(divBYThree({ number: 15 })).toBe('15 is divisible by three.')
```

Comment:
Good job constructing your test. Remember, using the variables given in your arguments in the expect statements is best practice